woocommerce/woocommerce
# Changelogger: Consolidate JS packages' use of composer into one

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/38989



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. `pnpm install`
2. Check changelog CLI functions as it normally does
```
pnpm --filter=woocommerce run changelog add
pnpm --filter=@woocommerce/currency run changelog add
```
3. See the `Add changelog to PR` action pass and resulting changefiles with every touched project



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Consolidate Jetpack Changelogger dependency into single tool.

#### Comment 

</details>
